contentcuration/contentcuration/views/internal.py

```diff
@@ -216,9 +226,19 @@ def api_commit_channel(request):
         if not data.get('stage'):
             try:
                 activate_channel(obj, request.user)
+                # Prepare change event indicating new root_id
+                event = generate_update_event(channel_id, CHANNEL, {
+                    "root_id": obj.main_tree.id,
+                    "staging_root_id": None,
+                })
```

Comment:
@rtibbles Should we be sending any more info here as part of this UPDATE event?
count? updated? modified? name? --- theoretically anything could have changed on the channel between two chef runs (so wondering if we have to send all that info in this UPDATE event, or just send the new `root_id` and Studio frontend will know it needs to GET the updated info).